website/blog/2021-09-01-preparing-your-app-for-iOS-15-and-android-12.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Preparing Your App for iOS 15 and Android 12
+author: Samuel Susla
+authorTitle: Software Engineer on React Native
+authorURL: https://twitter.com/SamuelSusla
+authorImageURL: https://avatars.githubusercontent.com/u/1733610?v=4
+authorTwitter: SamuelSusla
+tags: [announcement]
```

Comment:
I'm not sure if this post should be tagged as announcement, it feels a bit more like guide. 

For the similar post in the past tag `engineering` has been used:
* https://reactnative.dev/blog/tags

Reply:
I changed the tag to engineering. Thank you.